libraries/apollo-api/api/apollo-api.api

```diff
@@ -1090,40 +1097,14 @@ public abstract interface class com/apollographql/apollo3/api/json/JsonWriter :
 	public abstract fun value (Z)Lcom/apollographql/apollo3/api/json/JsonWriter;
 }
 
-public final class com/apollographql/apollo3/api/json/MapJsonReader : com/apollographql/apollo3/api/json/JsonReader {
-	public static final field Companion Lcom/apollographql/apollo3/api/json/MapJsonReader$Companion;
+public final class com/apollographql/apollo3/api/json/MapJsonReader : com/apollographql/apollo3/api/json/MapishJsonReader {
```

Comment:
Pretty sure this is a breaking change we can't backport to 3.x. Are you ok living in 4.x land (`main`) for now? We were planning to start releasing alphas in late April but we could do a `-dev.1` if needed.

Reply:
Also +1 on the naming 😃 !

Reply:
😄 I cribbed it from Kotlin which internally has an `isArrayish()`.

Yes, we're fine waiting for 4.x. We don't plan on releasing anything until June-ish and can work off of SNAPSHOTs until then.